model_test.go

```diff
@@ -286,8 +286,12 @@ func NewRoleManager() rbac.RoleManager {
 	return &testCustomRoleManager{}
 }
 func (rm *testCustomRoleManager) Clear() error { return nil }
```

Comment:
Why the below two functions are expanded except this one?

Reply:
Either `goimports` or my IDE expanded the others. I did not manually expand any of those.

Reply:
OK. Strange..